module/VuFindSearch/src/VuFindSearch/Backend/Blender/Response/Json/RecordCollection.php

```diff
@@ -374,6 +374,19 @@ protected function mapFacetValues(array $collections, array $settings): array
                 }
             }
         }
+
+        foreach ($settings['Mappings'] as $backendId => $mappings) {
+            if (($collections[$backendId] ?? false) &&
+                ($ignore = $mappings['Ignore'] ?? '') &&
```

Comment:
Maybe `$mappings['Ignore'] ?? false`?

Reply:
If we're not using `$ignore` elsewhere, we definitely don't want/need to create an unused variable here.

Reply:
Yes that is better. I failed to realize it after changing the code.